package/MDAnalysis/core/groups.py

```diff
@@ -155,6 +213,7 @@ def __getitem__(self, item):
             if isinstance(item, list) and item:  # check for empty list
                 # hack to make lists into numpy arrays
                 # important for boolean slicing
+                # TODO: what about tuples?
```

Comment:
Ah, good point, I'm cool with this. We'll have to write it down somewhere in the docs, though.